.github/workflows/ci.yml

```diff
@@ -11,16 +11,29 @@ jobs:
   coding-guidelines:
     name: Coding Guidelines
     runs-on: ubuntu-latest
-
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.4']
```

Comment:
Only 7.4 here?

Reply:
Yes, because it's to run the coding-guidelines (php-cs-fixer), I don't think we have to run it for all php versions.

Reply:
I removed the matrix strategy for non tests jobs.

Reply:
It makes sense 👌🏼👌🏼